how-to/editor-in-chief-guide.md

```diff
@@ -90,9 +90,18 @@ organization.
 
 Once the package initial checks are complete, and it is determined that
 the package is in scope for pyOpenSci review, the Editor in Chief will assign an
-editor to the review issue. The editor will begin the process of finding reviewers
-for the package. [Check out the editor guide for more information on this process](editors-guide.md)
-
+editor to the review issue.
+This may involve finding a new editor
+as described in the [onboarding guide](onboarding-guide.md).
+If you as Editor in Chief do recruit a new editor,
+be sure to complete all the onboarding steps described
+[here](onboarding-a-new-editor) so that the new editor
+has everything they need to manage the review,
+such as GitHub permissions and access to the relevant channels
+in the pyOpenSci Slack team.
```

Comment:
this is great too - can we be specific?

so they need to be added to `private-editorial-team` channel in slack as well?